components/slider/index.css

```diff
@@ -427,6 +472,8 @@
 
 	font-size: var(--mod-slider-font-size, var(--spectrum-slider-font-size));
 	line-height: var(--mod-line-height-100, var(--spectrum-line-height-100));
+	font-weight: var(--mod-slider-label-font-weight, var(--spectrum-slider-label-font-weight));
+	font-style: var(--mod-slider-label-font-style, var(--spectrum-slider-label-font-style));
 
```

Comment:
Should we add the `default-font-family` token here as well? Looks like most of the others are here, except for that one.